website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -5,19 +5,27 @@ description: "You can set up your Azure DevOps by creating a Microsoft Entra ID
 sidebar_label: "Set up Azure DevOps"
 ---
 
-<Snippet path="available-enterprise-tier-only" />
+# Set up Azure DevOps <Lifecycle status="enterprise" />
 
-## Overview
+## Service principal overview
+
+:::note
+
+The instructions in this section are for creating a service principal app. This feature is being slowly rolled out to dbt Cloud accounts. If you need to configure dbt Cloud and Azure DevOps but don't see the **Service principal** option in your account settings, use the [**Service user**](#set-up-service-user) configuration for now. You can easily [migrate to a service principal](#migrate-to-service-principal) when it is available for your account.
+
+:::
+
+To use dbt Cloud's native integration with Azure DevOps, an account admin needs to set up an Microsoft Entra ID app as a service principal. We recommend setting up a separate [Entra ID application than used for SSO](/docs/cloud/manage-access/set-up-sso-microsoft-entra-id).
+
+The application's service principal represents the Entra ID application object. Whereas a service user represents a real user in Azure with an Entra ID (and an applicable license), the service principal is a secure identity used by an application to access Azure resources unattended. The service principal authenticates with a client ID and secret rather than a username and password (or any other form of user auth). Service principals are the [Microsoft recommended method](https://learn.microsoft.com/en-us/entra/architecture/secure-service-accounts#types-of-microsoft-entra-service-accounts) for authenticating apps. 
```

Comment:
```suggestion
The application's service principal represents the Entra ID application object. While a "service user" represents a real user in Azure with an Entra ID (and an applicable license), the "service principal" is a secure identity used by an application to access Azure resources unattended. The service principal authenticates with a client ID and secret rather than a username and password (or any other form of user auth). Service principals are the [Microsoft recommended method](https://learn.microsoft.com/en-us/entra/architecture/secure-service-accounts#types-of-microsoft-entra-service-accounts) for authenticating apps. 
```